ztest/ztest.go

```diff
@@ -273,6 +273,7 @@ type ZTest struct {
 	OutputFlags string `yaml:"output-flags,omitempty"`
 	ErrorRE     string `yaml:"errorRE,omitempty"`
 	Vector      bool   `yaml:"vector"`
+	VectorOnly  bool   `yaml:"vector_only"`
```

Comment:
You're not adding any tests that use this.
```suggestion
```

Reply:
whoops